src/App.tsx

```
@@ -7,91 +7,117 @@ import { useChat } from './hooks/useChat';
+					/>
+				</div>
+
+				{/* Main Content with Sidebar */}
+				<div className="flex-1 flex bg-white/5 backdrop-blur-sm border border-white/10 rounded-2xl overflow-hidden">
+					{/* Left Sidebar for Last Question */}
+					<div className="w-1/4 bg-white/10 border-r border-white/10 p-4 flex flex-col items-start justify-start">
+						<h2 className="text-sm font-semibold text-gray-300 mb-2">Last Question</h2>
+						{lastUserMessage ? (
+							<div
+								className="text-gray-100 text-base break-words"
+								dangerouslySetInnerHTML={{ __html: lastUserMessage.content }}
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)